statik/database.py

```diff
@@ -418,6 +418,14 @@ def __init__(self, model=None, session=None, **kwargs):
                     "Attempting to look up primary keys for ManyToMany " +
                     "field relationship: %s", self.field_values[field_name]
                 )
+
+                # check if non-string items are present
+                if not all(isinstance(item, str) for item in self.field_values[field_name]):
+                    logger.warning("Non-string values in array (field: %s, model: %s): %s",
+                                    field_name, self.model.name, self.field_values[field_name])
+                    # convert all items to strings
+                    self.field_values[field_name] = list(map(str, self.field_values[field_name]))
```

Comment:
again `str` will cause problems on Python 2.7

conversion is beyond the scope of your task.  a warning will suffice.

dont worry about doing this conversion, especially if fixing this line is not trivial due to unicode.

very likely, this warning will be upgraded to an error, after the next release, so conversion wont be useful then.